src/models/user.js

```diff
@@ -70,6 +82,7 @@ class User {
   }
   set firstName(newFirstName) {
     if (newFirstName) {
+      this._isDirty = true;
```

Comment:
[lidea/question] Does it make sense `_isDirty` to be a function and "calculate" returned value dynamically?